docs/reference/analysis/analyzers.asciidoc

```diff
@@ -1,71 +1,148 @@
 [[analysis-analyzers]]
 == Analyzers
+* <<configuring-analyzers, Configuring Analyzers>>
+* <<default-analyzers, Configuring Default Analyzers>>
+* <<aliasing-analyzers, Aliasing Analyzers>>
+* <<backwards-compatibility, Backward Compatibility>>
 
-Analyzers are composed of a single <<analysis-tokenizers,Tokenizer>> 
-and zero or more <<analysis-tokenfilters,TokenFilters>>. The tokenizer may 
-be preceded by one or more <<analysis-charfilters,CharFilters>>.
-The analysis module allows you to register `Analyzers` under logical
-names which can then be referenced either in mapping definitions or in
-certain APIs. 
+Analyzers break text down into component parts that can be efficiently looked  
+up when you search your data. Analyzers process text fields during indexing 
+to determine what tokens are added to the index. When you perform a 
+full-text search, analyzers process the query string so that the
+search terms are represented in the same form as the indexed data.
 
-Elasticsearch comes with a number of prebuilt analyzers which are
-ready to use.  Alternatively, you can combine the built in
-character filters, tokenizers and token filters to create 
-<<analysis-custom-analyzer,custom analyzers>>.
+An analyzer contains one <<analysis-tokenizers,tokenizer>> and optional 
+<<analysis-charfilters,character>> and <<analysis-tokenfilters,token>> 
+filters. 
+
+image:images/AnalysisChain.png[]
+
+If present, _character filters_ preprocess the incoming text before
+passing it on to the tokenizer. For example, you can use character
+filters to strip out HTML tags or replace a particular character or 
+pattern of characters. 
+
+The _tokenizer_ splits the individual terms in the text apart on boundaries
+such as whitespace and punctuation and outputs a set of tokens that can be
+indexed. 
+
+If present, _token filters_ postprocess the set of tokens output by the 
```

Comment:
I'd use a definition list here, as in:

    Character filters::

    If present, _character filters_ ...